src/model.rs

```diff
@@ -18,6 +18,8 @@ rewrite.
 */
 
 use errors::*;
+use ex::ExCrate;
+use result::OptionResultExt;
```

Comment:
I'm not sure why rust is complaining that this is unused, since it also complains if I remove it.